doc/chips/10.rst

```diff
@@ -250,6 +250,78 @@ block while the rules for Phase 2 may be applied to the second, without a
 constant check to the location of the ``.init()`` call which is the linchpin for
 syntax 1.
 
+Child's fields could depend on parent's fields
+**********************************************
+
+We also considered allowing child's fields to be initialized based on
+parent's fields. For example, the child could control where, in its Phase 1,
+the parent's Phase 1 is invoked, not necessarily as the last step.
+
+The following example shows that based on the second syntax above.
+It also shows the option of having the child control when the parent's Phase 2
+is invoked.
```

Comment:
Please demonstrate two concepts using two distinct examples.  The second example may rely on the first.  I also object to the phrasing of these two sentences, they do not match the tone and formality of the rest of the document - I will get back to you with suggestions for how to write this in a more formal manner.